certified-connectors/VirusTotal/readme.md

```diff
@@ -0,0 +1,15 @@
+
+Analyze suspicious files and URLs to detect types of malware, automatically share them with the security community.
+
+
+## Pre-requisites
+
+To use this integration, you need to have a VirusTotal account. Follow steps on https://www.virustotal.com/ to get your API Key.
```

Comment:
- Replace `https://www.virustotal.com/` with `https://developers.virustotal.com/v3.0/reference#getting-started`.
- Replace `Key` with `key`.
- Remove space between `[]` and `()` in like 15 to make it a valid Markdown link.

Reply:
done